### ISSUE TITLE
example on readme not working

### ISSUE THREAD

**Issue by mantzaris** [JuliaStats/Loess.jl#31]
1. The ys data needed the broadcast operator; ys = sin.(xs) .+ 0.5 * rand(100)
2. In attempting to use the *loess* model construction method, an error is returned even after 'using' all relevant packages; 
```
model = loess(xs, ys)
ERROR: cannot assign variable Bootstrap.model from module Main
Stacktrace:
 [1] top-level scope at none:0
```

**Comment by tpapp** [comment]
I fixed the broadcasting in #32, and can't reproduce the second error.

**Comment by tlnagy** [comment]
You likely ran `using Bootstrap` which imported the `model` function into your namespace and then that conflicted with you setting the `model` variable with `model = loess(xs, ys)`
